utils/glue_generator_src/glue_generator.cpp

```diff
@@ -502,6 +504,12 @@ void generateBody(istream& locatedVars, ostream& glueVars, md5_byte_t digest[16]
         uint16_t pos1;
         uint16_t pos2;
         findPositions(iecVar_name, &pos1, &pos2);
+
+        if (pos2 > 7) {
+            cout << "Sub-position of variable " << iecVar_name << " must be in the range 0 to 7";
+            return -1;
```

Comment:
I think it's better to throw an exception here - otherwise CMake will not fail. Something like:

```
const std::string e_discription = std::string("Sub-position of variable ") + iecVar_name + " must be in the range 0 to 7";
throw std::out_of_range(e_discription);
```

and an `#include <stdexcept>` works for me.